test/OpenTelemetry.Tests/EventSourceTest.cs

```diff
@@ -28,12 +28,6 @@ public void EventSourceTest_InstrumentationEventSource()
             EventSourceTestHelper.MethodsAreImplementedConsistentlyWithTheirAttributes(InstrumentationEventSource.Log);
         }
 
-        [Fact]
-        public void EventSourceTest_OpenTelemetryApiEventSource()
-        {
-            EventSourceTestHelper.MethodsAreImplementedConsistentlyWithTheirAttributes(OpenTelemetryApiEventSource.Log);
-        }
-
         [Fact]
```

Comment:
This class was split into two, and only the OpenTelemetry.Api test needed to be moved.